README.md

```diff
@@ -51,6 +51,14 @@ $ tadpoles-backup stat
 $ tadpoles-backup backup "/a/directory/on/your/machine/"
 ```
 
+With docker:
+
+```
+docker build -t tadpoles .
+mkdir ~/tadpoles_bk
+docker run -it -v$HOME/tadpoles_bk:/usr/src/app/data --rm tadpoles bin/tadpoles-backup backup data
```

Comment:
it would be great not to have to specify `-it` just to input the user/pass. I will work on some enhancements to improve the tool for automated workflows. I will track these changes [HERE](https://github.com/leocov-dev/tadpoles-backup/issues/36).